.gitignore

```diff
@@ -40,3 +40,6 @@ testem.log
 # System Files
 .DS_Store
 Thumbs.db
+
+# Translations
+apps/*/public/locales/*
```

Comment:
If I'm not mistaken, the recent work merged on `hublo-front` moved to a "commit the translation" convention.

Maybe we should align ? Or this is the config we aim for when everything will be moved to `nx` ?

Reply:
Actually yes they should be committed. But this is not something we agreed on with @ArthurGoupil . Right now, I believe locales are not commited, just pulled before build.

I think it would be better to have them committed to ensure devs don't need to always run `setupTranslations`.

What do you think @PupoSDC @balzdur @ArthurGoupil ?

Reply:
I would suggest to commit the translations. I've seen in the past poor translations being caught in PR reviews by more grammer oriented reviewers, so it's an extra safety mechanism.